views/install.sh.erb

```diff
@@ -439,8 +447,13 @@ install_file() {
   echo "Installing $project $version"
   case "$1" in
     "rpm")
-      echo "installing with rpm..."
-      rpm -Uvh --oldpackage --replacepkgs "$2"
+      if test "x$platform" = "xnexus" || test "x$platform" = "xios_xr"; then
+        echo "installing with yum..."
+        yum install -yv "$2"
```

Comment:
Uhm, okay, I'll just quietly slink back into the darkness now...

Ignore any sounds that may appear to be my head smacking into hard objects...